internal/pkg/util/errno_windows.go

```diff
@@ -0,0 +1,22 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License 2.0;
+// you may not use this file except in compliance with the Elastic License 2.0.
+
+//go:build windows
+
+package util
+
+import (
+	"errors"
+	"syscall"
+)
+
+// FilterErrno filters out Errno = 0 which indicates success
+func FilterErrno(err error) error {
```

Comment:
This is now only being used in testing, I think it would be better to just remove it.